technicalreport/evaluation.tex

```diff
@@ -13,92 +13,194 @@ \subsubsection{Kriterien}
-        Proband 1                & 1                        & 1                        \\
-        Proband 2                & 1                        & 1                        \\
-        Proband 3                & 1                        & 1                        \\
-        Proband 4                & 1                        & 1                        \\
-        Proband 5                & 1                        & 1                        \\
-        \textbf{Europaplatz}     &                          & 1                        \\
-        Proband 1                & 1                        & 1                        \\
-        Proband 2                & 1                        & 1                        \\
-        Proband 3                & 1                        & 1                        \\
-        Proband 4                & 1                        & 1                        \\
-        Proband 5                & 1                        & 1                        \\
-        \textbf{Resultat}        & \textbf{x}               & \textbf{y}                       
+        Proband 1                & 4                        & 3                        \\
+        Proband 2                & 4                        & 3                        \\
+        Proband 3                & 3                        & 4                        \\
+        Proband 4                & 2                        & 3                        \\
+        Proband 5                & 3                        & 4                        \\
+        \textbf{Bahnhofplatz Bern} &                        &                          \\
+        Proband 1                & 4                        & 3                        \\
+        Proband 2                & 3                        & 4                        \\
+        Proband 3                & 4                        & 2                        \\
+        Proband 4                & 3                        & 4                        \\
+        Proband 5                & 2                        & 3                        \\
+        \textbf{Resultat}        & \textbf{3.33}               & \textbf{3.4}                       
     \end{tabular}
 \end{table}
 
-\paragraph{Resultat und Schlussfolgerung}\label{result:Resultat und Schlussfolgerung}~\\
+\subsubsection{Auswertung und Schlussfolgerung}
+\label{result:Auswertung und Schlussfolgerung}
+
+Die einzelnen Werte sind in Tabelle \ref{table:Resultat Evaluation Flächen-Algorithmus} zusammen gefasst und gewichtet. Es zeigt sich, dass der Visibility Graph knapp besser abschneidet, der Unterschied ist allerdings sehr gering und nicht eindeutig.
+
+Bei der Bewertung der Natürlichkeit durch Probanden ergaben sich einige Erkenntnisse. So bewerteten einige Probanden eine direkte Route mit etwas ``scharfen'' Ecken - wie sie der Visibility Graph erzeugt - natürlicher, während andere die etwas kurvigeren Routen des Spiderweb Graphs bevorzugten.
```

Comment:
hier wäre noch eine Relativierung der Befragung sinnvoll. Da es ja nur 5 Personen sind, müsste man es schon noch in de Kontext setzen, dass man einen Eindruck gewinnen wollte